user_tools/src/spark_rapids_tools/tools/qualx/model.py

```diff
@@ -174,19 +172,23 @@ def predict(
         preds['y'] = y
     preds_df = pd.DataFrame(preds)
 
+    label = get_label()  # Duration, duration_sum
     select_columns = [
         'appName',
         'appId',
         'appDuration',
         'sqlID',
         'scaleFactor',
-        'Duration',
+        label,
         'fraction_supported',
         'description',
     ]
     if 'split' in cpu_aug_tbl:
         select_columns.append('split')
 
+    if label not in cpu_aug_tbl:
+        raise ValueError(f'{label} column not found in input data')
```

Comment:
I think my original comment on this was wrong since in the case of prediction (i.e. no training) there wouldn't be a label comment.